.buildkite/pipeline-resource-definitions/scripts/fix-location-collection.ts

```diff
@@ -36,15 +36,13 @@ async function main() {
 
   const preamble = locationFileLines.slice(0, 1);
 
-  // eslint-disable-next-line @kbn/eslint/no_unsafe_js_yaml
```

Comment:
Looks like the upgrade to 4 isn't done yet, so these are going fail - https://github.com/elastic/kibana/pull/190678

Reply:
I got those changes from merging main. Should I manually revert these change?

Reply:
and my editor is still yelling at me
![image](https://github.com/user-attachments/assets/495c0575-e5b9-4122-b91d-e86c93109044)

Reply:
They don't need to be reverted I guess - https://github.com/elastic/kibana/blob/main/.buildkite/pipeline-resource-definitions/scripts/fix-location-collection.ts#L10, it's removed on main too.  GitHub is showing us as codeowners of these changes but we shouldn't be needed to review already existing changes.  Maybe merging in main again will resolve it.

For your editor:  can you reproduce this by running `node scripts/eslint .buildkite/pipeline-resource-definitions/scripts/fix-location-collection.ts`?  If not the editor is bugged and I would try restarting it.

Reply:
After merging main:
```
kevin@Kevins-MacBook-Pro ~/workspace/kibana (rca/esql-item)$ node scripts/eslint .buildkite/pipeline-resource-definitions/scripts/fix-location-collection.ts

/Users/kevin/workspace/kibana/.buildkite/pipeline-resource-definitions/scripts/fix-location-collection.ts
  39:30  error  Use `safeLoad` instead of `load` and `safeDump` instead of `dump` from `js-yaml`  @kbn/eslint/no_unsafe_js_yaml
  46:31  error  Use `safeLoad` instead of `load` and `safeDump` instead of `dump` from `js-yaml`  @kbn/eslint/no_unsafe_js_yaml

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
Ah my bad - I thought the eslint-ignore was on line 10.  It's not.  Going back to my original statement, the ignore lines are supposed to be there and should be reverted.  They exist on main.

Reply:
https://github.com/elastic/kibana/blob/d3fdb7d8d6d11fc83dde384b524066954880e28f/.buildkite/pipeline-resource-definitions/scripts/fix-location-collection.ts#L39-L47

Reply:
Should be reverted